cfme/tests/containers/test_containers_simple_search.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from itertools import product
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import Table
+from cfme.web_ui import search
+
+from utils import testgen
+from utils.version import current_version
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+PROJECTS_SEARCH_STRINGS = ['*', 'infra', '$', 'management-infra']
+ROUTES_SEARCH_STRINGS = ['*', 'front', '$', 'frontend']
+table_el = '//table[contains(@class, ' \
+           '"table table-striped table-bordered table-hover table-selectable")]'
+
+# CMP-9871
+
+
+@pytest.mark.parametrize('projects_search_strings', product(PROJECTS_SEARCH_STRINGS))
+def test_projects_search(projects_search_strings):
+    """ Projects summary page - Search bar
+
+           This test checks Search bar functionality on the Projects summary page
+
+           Steps:
+               * Goes to Containers Projects page
+               * Inserts: Irregular symbol, '*' character, full search string, partial search string
+               * Verify proper results
+           """
+    if ''.join(projects_search_strings) == '*':
```

Comment:
This and the function below (at first glance) look identical, does it make sense to apply some DRY here and make this a function where "containers_projects" is an argument to the function?